R/add_creator_id.R

```diff
@@ -0,0 +1,67 @@
+#' add_creator_id
+#'
+#' This function allows you to add an ORCID or reference ID to a creator in EML.
```

Comment:
I think you want a blank line between each unique roxygen section.  Between param and keywords, keywords and details, etc.

Reply:
doesn't seem to be necessary based on this example: https://hilaryparker.com/2014/04/29/writing-an-r-package-from-scratch/
but I'll change it if it throws errors.